static/js/components/pressure.vue

```diff
@@ -137,6 +137,21 @@
         }
         return maxPressure;
       },
+      dataAvailable: function() {
+        for (let bin in this.rawData) {
+          for (let proposal in this.rawData[bin]) {
+            if (this.rawData[bin][proposal] > 0) {
+              return true;
+            }
+          }
+        }
+        return false;
+      }
+    },
+    created: function() {
+      this.fetchData();
+    },
+    methods: {
       toSiteNightData: function() {
```

Comment:
If `toSiteNightData` is now a method, doesn't that break where it is used?

Reply:
ah I guess I should add () where we call the method...